packages/contracts-bedrock/package.json

```
@@ -15,12 +15,14 @@
   ],
   "scripts": {
     "bindings": "cd ../../op-bindings && make",
-    "build:forge": "forge build",
     "build:with-metadata": "FOUNDRY_PROFILE=echidna yarn build:forge",
     "build:differential": "tsc scripts/differential-testing.ts --outDir dist --moduleResolution node --esModuleInterop",
     "build:fuzz": "go build -o test-case-generator test-case-generator/cmd/fuzz.go",
-    "prebuild": "yarn ts-node scripts/verify-foundry-install.ts",
-    "build": "hardhat compile && yarn autogen:artifacts && yarn build:ts && yarn typechain",
+    "build": "npx nx build:ts && npx nx typechain",
```

Why `npx nx` rather than `yarn nx`?

---

in general the difference is npx will download the library if you don't already have it installed.  But it first detects if you have it installed already.  So in this case there isn't a difference.   

It can still be nice though.  For example, if a tool uses forge and doesn't need node modules you can still npx do stuff without needing to do a full yarn install.

---

I suppose also that this PR moves `yarn` closer to not much more than a wrapper around `nx`, so using `npx` moves us closer to being able to remove `yarn`, yeah?

Would it eventually work to rewrite this as follows and then call it with `nx build`?
```suggestion
    "build": "nx build:ts && nx typechain",
```

---

Yea using npx makes it less tied to yarn that is true.  I suggest anybody using nx `npm install nx --global` so they never have to type either yarn or tsx in their dev env though.